journal.py

```diff
@@ -11,107 +10,136 @@
 from pyramid.session import SignedCookieSessionFactory
 from pyramid.view import view_config
 from waitress import serve
-from pyramid.events import NewRequest, subscriber
-from pyramid.httpexceptions import HTTPFound, HTTPInternalServerError, HTTPForbidden
+# from pyramid.events import NewRequest, subscriber
+from pyramid.httpexceptions import HTTPFound, \
+    HTTPInternalServerError, HTTPForbidden
 from pyramid.authentication import AuthTktAuthenticationPolicy
 from pyramid.authorization import ACLAuthorizationPolicy
 from cryptacular.bcrypt import BCRYPTPasswordManager
 from pyramid.security import remember, forget
 from jinja2 import Environment, PackageLoader
-
-env = Environment(loader=PackageLoader('journal', 'templates'))
-
-here = os.path.dirname(os.path.abspath(__file__))
-
-DB_SCHEMA = """
-CREATE TABLE IF NOT EXISTS entries (
-    id serial PRIMARY KEY,
-    title VARCHAR (127) NOT NULL,
-    text TEXT NOT NULL,
-    created TIMESTAMP NOT NULL
+import sqlalchemy as sa
+from zope.sqlalchemy import ZopeTransactionExtension
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import (
+    scoped_session,
+    sessionmaker,
 )
-"""
 
-INSERT_ENTRY = """
-INSERT INTO entries (title, text, created) VALUES (%s, %s, %s)
-"""
 
-DB_ENTRIES_LIST = """
-SELECT id, title, text, created FROM entries ORDER BY created DESC
-"""
+env = Environment(loader=PackageLoader('journal', 'templates'))
 
-SELECT_ID = """
-SELECT * FROM entries WHERE id=%s
-"""
+here = os.path.dirname(os.path.abspath(__file__))
 
-READ_ENTRY = """
-SELECT id, title, text, created FROM entries WHERE id = %s
-"""
+# DB_SCHEMA = """
+# CREATE TABLE IF NOT EXISTS entries (
+#     id serial PRIMARY KEY,
+#     title VARCHAR (127) NOT NULL,
+#     text TEXT NOT NULL,
+#     created TIMESTAMP NOT NULL
+# )
+# """
+
+# INSERT_ENTRY = """
+# INSERT INTO entries (title, text, created) VALUES (%s, %s, %s)
+# """
+
+# DB_ENTRIES_LIST = """
+# SELECT id, title, text, created FROM entries ORDER BY created DESC
+# """
+
+# SELECT_ID = """
+# SELECT * FROM entries WHERE id=%s
+# """
+
+# READ_ENTRY = """
+# SELECT id, title, text, created FROM entries WHERE id = %s
+# """
+
+# UPDATE_ENTRY = """
+# UPDATE entries SET (title, text) = (%s, %s) WHERE id=%s
+# """
+
+# NEW_ENTRY = """
+# SELECT * FROM entries ORDER BY created DESC LIMIT 1
+# """
```

Comment:
Delete stuff rather than commenting it out.  Trust in version control to keep hold of it for you.